lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -196,6 +211,9 @@ class CategoryRenderer {
     if (expandable) {
       const chevronEl = summaryInnerEl.appendChild(this._createChevron());
       chevronEl.title = Util.UIStrings.auditGroupExpandTooltip;
+      if (opts.expandByDefault) {
```

Comment:
seems like this is ripe for some cleanup like our boolean traps over in page load.

WDYT about an enum instead of the two booleans `type ExpansionBehavior = 'open-permanent' | 'closed-expandable' | 'open-expandable'` or something like that?